### DISCUSSION TITLE
Zwave Route Map

### DISCUSSION THREAD

**Discussion by Siburanjan** [zwave-js/zwave-js#7443] [Feature Requests]
### Description

I want an API such that calling that API gives me the route from the controller node to the all other end nodes

### Do you have a solution in mind?

I was polling the neigbour nodes and mapping it but its creating too much load on the gateway. Suggest me the best way.

**Comment by Siburanjan** [comment]
@robertsLando please help me out with this its quite urgent requirement

**Comment by Siburanjan** [comment]
Also I need an API to fetch the signal strength of the complete zwave network

**Comment by AlCalzone** [comment]
Please read the answers I gave you 24 days ago:

https://github.com/zwave-js/node-zwave-js/discussions/7359#discussioncomment-11147133
